core/app/models/spree/state.rb

```diff
@@ -6,7 +6,7 @@ class State < Spree::Base
     validates :country, :name, presence: true
 
     def self.find_all_by_name_or_abbr(name_or_abbr)
```

Comment:
Completely agree.  I'll switch to `with_name_or_abbr` with a deprecation for the old method.